### PR TITLE
feature #10 tests done successfully and snapshots added

### LINKED ISSUE #10: Feat: Manual testing of Sign up and Sign in feature
**What is your issue related to ?**

- [ ] Code
- [ ] Designing
- [x] Documentation
- [x] Testing/Research
- [ ] Others[please specify]

**Description**
Download the apk[[release version](https://github.com/Code-Sauce-Official/FitMate/releases/tag/1.0)] and test all different scenarios for sign up and sign in.

An md file with all the test scenarios you covered. Positive and negative test cases as expected. Each test scenario should be documented as below:

```
* _Test1 Description_  
     _Screenshot/gif_:  
     _Expected Result_:  
     _Actual Result_:

* _Test2 Description_  
     _Screenshot/gif_:  
     _Expected Result_:  
     _Actual Result_:  
     ...  
```
**Additional context**
Add your markdown(md) file in [Testing folder](https://github.com/Code-Sauce-Official/FitMate/tree/develop/Documentation/Testing)
`If you see issues or unexpected behavior during testing feel free to open an issue`

### PR DESCRIPTION
**Pull request by KishkinJ10**
## Description

<This PR is regarding a feature request #10 in which test have been done successfully to check sign IN and sign-up functionality and screenshots have been added>

## Related Issue

Fixes #10  

## Type of Contribution:

- [ ] Coding
- [x] Testing
- [ ] Research
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] I have read README.md and CONTRIBUTION.md

- [x] My code/design follows the style/guidelines of this project.
- [x] I have performed a self-review of my own code/design/documentation.
- [ ] I have commented my code, particularly in hard-to-understand areas(or gave a proper description about the design).

### CODE REVIEW

**Comment by SHITIZ-AGGARWAL** [comment]
@KishkinJ10 Use the url of photos in

**Review by SHITIZ-AGGARWAL — CHANGES_REQUESTED** [review]
Also please give space between different lines

**Comment by KishkinJ10** [comment]
Sir please check the changes made :)

**Review by Acash512 — CHANGES_REQUESTED** [review]
Implement the following:
- Errors set on textfields like Email cannot be empty, constraints on password in Sign Up(like min 8 digits, special character etc)
- Tests for Resend verification email
- Tests for Forgot Password

**Comment by KishkinJ10** [comment]
Sir please check the changes.

File: Documentation/Testing/SignUpTesting.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     <img src="TestSignUpAndSignIn_Snapshots/Empty_Fields_Fitmate.jpg" width="300"/><br>
+     _Expected Result_: Email and Password cannot be empty<br>
+     _Actual Result_:  Positive
+     
+* _Test6 Password Constraints in sign Up_  
+     _Screenshot/gif_:  
+     <img src="TestSignUpAndSignIn_Snapshots/Password_Conatraint1_FitMate.jpg" width="300"/><br>
+     <br>
+    <img src="TestSignUpAndSignIn_Snapshots/Password_Constraint2_FitMate.jpg" width="300"/><br>
+     _Expected Result_:  Password must contain atleast one special character and minimum 8 digits<br>
```
Comment (Acash512) [inline Documentation/Testing/SignUpTesting.md:37]: Password must also contain a letter and a digit

**Review by Acash512 — COMMENTED** [review]
Also add tests for Confirm Password, also if email has already been verified a toast saying "Email address already verified" is displayed on clicking Resend Verification Email, for forgot password, entering email address is mandatory, a toast is shown if email is empty

**Comment by SHITIZ-AGGARWAL** [comment]
image

@KishkinJ10 Use the URL properly

**Comment by SHITIZ-AGGARWAL** [comment]
@KishkinJ10 While working on testing issues, try to cover every possible test you can think of . The main motive is to see whether the feature is working positively for each and every scenario. **TRY CRASHING THE APP**

You can note down every possibility first and then make test cases based on that, this will help you to cover more scenarios.

**Comment by KishkinJ10** [comment]
@SHITIZ-AGGARWAL @Acash512  Sir I have added all the test which I could have thought of , sir mostly all the tests are covered . Sir please merge this PR :)

**Review by SHITIZ-AGGARWAL — APPROVED** [review]

**Review by yatharthmago01 — APPROVED** [review]

**Review by Acash512 — APPROVED** [review]

**Comment by SHITIZ-AGGARWAL** [comment]
@KishkinJ10 good work

**Comment by KishkinJ10** [comment]
@SHITIZ-AGGARWAL @yatharthmago01 @Acash512 Thank you sir for guiding me.

**Pull request merged**